### PR TITLE
update example to AltDriver v1.8.0

### PR DESCRIPTION
**Pull request by SandyEma9**

### CODE REVIEW

**Comment by SandyEma9** [comment]
@robert-altom @RobertPoienar @gomboskriszta15 @ReichEdvin

File: README.md [hunk 1]
```diff
@@ -1,14 +1,21 @@
 # Standalone Build with Python Tests
 
-This repository shows a few Java tests that use the page object model and AltUnityTester to test the Unity endless runner sample:
+This repository shows a few Python tests that use the page object model and AltTester to test the Unity endless runner sample:
```
Comment (ReichEdvin) [inline README.md:3]: AltTester Unity SDK

**Review by ReichEdvin — APPROVED** [review]
LGTM

**Review by iulianatodoran — APPROVED** [review]
Tested locally and LGTM

**Pull request merged**